discoverer/README.md

```diff
@@ -0,0 +1,38 @@
+SeaClouds Discoverer
+====================
+
+The SeaClouds Discoverer module provides information about cloud offering used by SeaCloudsPlatform.
+
```

Comment:
- how?
- what kind of information? 
- where are those information fetched from? 
- why? what problem does it solve?
- and so on ...